api.planx.uk/modules/send/email/index.ts

```diff
@@ -69,7 +69,9 @@ export async function sendToEmail(
     );
 
     return res.status(200).send({
-      message: `Successfully sent "Submit" email`,
+      message: `Successfully sent to email`,
+      inbox: sendToEmail,
+      govuk_notify_template: "Submit",
```

Comment:
Slightly tweaking API responses here so they offer a bit more context in the "response" row (will apply to future events only) :heavy_check_mark: